packages/info/doctor/mixin.core.js

```diff
@@ -21,32 +42,45 @@ class DoctorMixin extends Mixin {
   bootstrap() {
     const { doctor } = this;
 
-    return this.diagnose(doctor).then((results) =>
-      doctor.collectResults(
-        ...[].concat(...results.filter((result) => result !== undefined))
-      )
-    );
+    return Promise.resolve().then(() => {
+      doctor.getMode().then((mode) =>
```

Comment:
Do we already have a use case for this feature?

Reply:
One example is the port, where no feedback is required if the app is in `build`-mode. And I'm pretty sure we find cases, where the decision on emitting an error or a warning depends on whether the app is in `develop`- or (production-)`serve`-mode.